test/modules/encoders/test_text_encoder.py

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import pytest
+import torch
+from test.test_utils import assert_expected, set_rng_seed
+from torch import Tensor
+from torchmultimodal.modules.encoders.text_encoder import bert_text_encoder
+
+
+@pytest.fixture(autouse=True)
+def random():
+    set_rng_seed(4)
+
+
+class TestTextEncoder:
```

Comment:
`TestBertTextEncoder`? Also maybe rename this file to `test_bert_text_encoder`

Reply:
Agreed.